crypto/x509v3/v3_cpols.c

```diff
@@ -413,7 +413,8 @@ static void print_notice(BIO *out, USERNOTICE *notice, int indent)
             num = sk_ASN1_INTEGER_value(ref->noticenos, i);
             if (i)
                 BIO_puts(out, ", ");
-            tmp = i2s_ASN1_INTEGER(NULL, num);
+            if ((tmp = i2s_ASN1_INTEGER(NULL, num)) == NULL)
+                return;
```

Comment:
@snhenson, Colleague of mine was able to hit the crash repeatably from w3m built against 1.0.2 by browsing www.digikam.org -> "Contact" -> "Mailing list subscription". I however never managed to trigger it.